doc/developers/contributing.rst

```diff
@@ -1112,3 +1112,32 @@ that implement common linear model patterns.
 
 The :mod:`sklearn.utils.multiclass` module contains useful functions
 for working with multiclass and multilabel problems.
+
+Estimator Tags
+--------------
+Scikit-learn introduced estimator tags in version 0.19.
+These are annotations of estimators that allow programmatic inspection of their
+capabilities, such as sparse matrix support, supported output types and
+supported methods.
+The estimator tags are a dictionary returned by the method ``_get_tags()``.
+These tags are used by the common tests and the ``check_estimator`` function to decide
+what tests to run and what input data is appropriate.
+
+The current set of estimator tags are:
+
+input_validation - whether the estimator does input-validation. This is only meant for stateless and dummy transformers!
+multioutput - whether a regressor supports multi-target outputs or a classifier supports multi-class multi-output.
+multilabel -  whether the estimator supports multilabel output
```

Comment:
So far I only added tags that were needed for the test to pass without special cases - actually I didn't need the sparse data one so far or the meta-estimator!